### PR TITLE
basic PDF rendering using native PDF viewer

### PR DESCRIPTION
**Pull request by dlockhart**
@omsmith, @jstefaniuk-d2l: seem reasonable? This is just super basic right now, next step will be to detect if the browser even supports native PDF rendering and if not, fall back to PDF.js.

### CODE REVIEW

**Comment by jstefaniuk-d2l** [comment]
LGTM

File: src/plugins/pdf/native.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+var React = require('react'), // eslint-disable-line no-unused-vars
```
Comment (omsmith) [inline src/plugins/pdf/native.js]: Does this put it into _jsx mode_ or something?

**Comment by omsmith** [comment]
LGTM, pending the 1 question

File: src/plugins/pdf/native.js (hunk 1 above)
Comment (dlockhart) [inline src/plugins/pdf/native.js]: Yeah, without it you can't do the JSX `<NativeViewer />` bit

Comment (omsmith) [inline src/plugins/pdf/native.js]: neato

**Comment by omsmith** [comment]
:+1:

**Pull request merged**